client/src/components/FeedbackForm/FeedbackModal.jsx

```diff
@@ -12,6 +13,9 @@ const MODAL_PROPTYPES = {
 };
 
 const FeedbackModal = React.memo(({ isOpen, toggle }) => {
+  const showUserGuide = useSelector(
+    (state) => state.workbench.config.showUserGuide
```

Comment:
One minor wording thing - think we could change `showUserGuide` to `hasUserGuide`? Might be more applicable to the effect it has on the frontend

Reply:
I agree, it reads better.